docs/index.rst

```diff
@@ -343,7 +343,7 @@ Most panels do not support any extra active features and need not be
 explicitly activated. However, some panels support an optional
 :attr:`~pyramid_debugtoolbar.panel.DebugPanel.is_active` state in which they
 will do some extra work. For example, the
-:class:`~pyramid_debugtoolbar.panels.performance.PerformanceDebugPanel``
+:ref:`PerformanceDebugPanel <panel_performance>`
```

Comment:
The autolink for this should have been:

    :class:`~pyramid_debugtoolbar.panels.performance.PerformanceDebugPanel`

however after correcting that, it linked to the api docs which do nothing. I changed this link to read the same, but now link to the panel docs.

Reply:
LGTM.